src/account/AccountPickScreen.js

```diff
@@ -10,6 +11,12 @@ import { Centerer, ZulipButton, Logo, Screen, ViewPlaceholder } from '../common'
 import AccountList from './AccountList';
 import { navigateToRealmScreen, switchAccount, removeAccount } from '../actions';
 
+const styles = StyleSheet.create({
+  accountList: {
+    flex: 0,
+  },
+});
```

Comment:
In general, performing the inverse of part of an aggregate operation before performing the aggregate operation itself is a good way to make code too intertwined and tangled to easily understand or modify. This is a last-resort kind of hack, and would need to be very well documented as such. (Also, I believe the correct value to prevent `flex` from being set  here would actually be the string `"unset"`. [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/unset).)

The ideal fix here is probably to do what I hinted at last time, and remove `flexed` from `Screen` – except with documentation, analysis, and some sort of fix so that [this bug](https://github.com/zulip/zulip-mobile/commit/026814edd7f4a3a19b9486bd3c01b20f91c52cd0#r31336633) doesn't recur.

Failing that, though, I'm pretty sure it should still be possible to adjust the layout of `AccountPickScreen` and/or `AccountList` so that the button isn't pushed off the screen, even without CSS hackery.